### PR TITLE
WoE: Symposium should handle tokens that get destroyed on use

### PR DESCRIPTION
**Pull request by strib**
Reported on discord.

### CODE REVIEW

File: server/game/cards/06-WoE/Symposium.js [hunk 1]
```diff
@@ -5,20 +5,25 @@ class Symposium extends Card {
...
+
         this.play({
             condition: (context) => context.game.creaturesInPlay.length > 0,
             target: {
                 controller: 'self',
                 cardType: 'creature',
                 gameAction: ability.actions.sequential([
-                    ability.actions.exalt(),
+                    ability.actions.exalt((context) => {
+                        this.wasToken = context.target.isToken();
```
Comment (strib) [inline server/game/cards/06-WoE/Symposium.js:17]: I suspect there is a better way of doing this, very open to suggestions!

Comment (romanogit) [reply server/game/cards/06-WoE/Symposium.js:17]: Probably we should use clone and Card's clone should consider tokenCard

Comment (strib) [reply server/game/cards/06-WoE/Symposium.js:17]: That sounds good.  But I tried looking at `context.preThenEvent` in this case, and `card === null` (so there is no clone made).  And I couldn't find any other child events or anything.  In `preThenContext` I can only find a clone of Symposium.  Any ideas about where to find the clone?

Comment (romanogit) [reply server/game/cards/06-WoE/Symposium.js:17]: I fixed createSnapshot to fix the cloned type, but I think I might also need to add a clone for istoken

**Comment by romanogit** [comment]
Used Cyberclone approach in #3496

**Pull request closed without merging**